src/Microsoft.Health.SqlServer/Features/Schema/SchemaInformation.cs

```diff
@@ -18,5 +18,14 @@ public SchemaInformation(int minimumSupportedVersion, int maximumSupportedVersio
         public int MaximumSupportedVersion { get; }
 
         public int? Current { get; set; }
+
+        /// <summary>
+        /// Checks if the version is not inside the full schema script.
+        /// </summary>
+        /// <param name="version">The schema version.</param>
+        public static bool DoesNotContainsVersionInSchema(int version)
```

Comment:
Nit: what about `SetsVersionInSchema` and `return version <= 5` (just to remove the negative from the boolean name)